browser-test/src/admin_application_statuses.test.ts

```diff
@@ -58,6 +58,34 @@ test.describe('view program statuses', () => {
     test('does not show edit note', async () => {
       expect(await ctx.adminPrograms.isEditNoteVisible()).toBe(false)
     })
+
+    test('does not show pagination when there is only 1 page of applications', async () => {
+      expect(await ctx.adminPrograms.isPaginationVisible()).toBe(false)
+    })
+
+    /* See trusted_intermediary.test.ts for more comprehensive pagination testing */
+    test('shows pagination if there are more than 10 applications', async () => {
+      const {page, adminPrograms, applicantQuestions} = ctx
+      // Apply to 10 more programs as a guest
+      for (let i = 0; i < 10; i++) {
+        await logout(page)
+
+        // Submit an application as a guest.
+        await applicantQuestions.clickApplyProgramButton(
+          programWithoutStatusesName,
+        )
+        await applicantQuestions.submitFromReviewPage()
+
+        // Navigate to the submitted application as the program admin.
+        await loginAsProgramAdmin(page)
+      }
+
+      // Navigate to the applications list
+      await adminPrograms.viewApplications(programWithoutStatusesName)
+
+      expect(await ctx.adminPrograms.isPaginationVisible()).toBe(true)
+      expect(page.locator('.usa-pagination__button:has-text("2")'))
```

Comment:
(non-blocking) Consider taking a screenshot here so we can have a visual diff with the pager.

Reply:
Oh if you need this test to have more than ten you may have an off by one on your loop only generating 10. Not sure if there is another application created elsewhere.

Reply:
Good idea to add a screenshot since all the other tests only have one page of applications.  

For the other concern, there's a single application that's being created in the `beforeAll` [here](https://github.com/civiform/civiform/blob/898fd655c2fc493f1505222535348a4a71030ab9/browser-test/src/admin_application_statuses.test.ts#L32).  I'll add a comment about that.

Reply:
Screenshot added and a comment about the beforeAll.